pywren/version.py

```diff
@@ -1,4 +1,8 @@
+from __future__ import print_function
 # we're following the version number guidelines
 # from https://www.python.org/dev/peps/pep-0386/
 
-__version__ = "0.1rc0"
+__version__ = "0.0.4rc3"
```

Comment:
This is actually because PyPI _never_ lets you reupload a release, so I had to keep incrementing fake numbers in order to get a new package up there to deploy